src/lm.jl

```diff
@@ -74,38 +81,50 @@ function nulldeviance(r::LmResp)
 end
 
 function loglikelihood(r::LmResp)
```

Comment:
Actually I've found the correct definition: it's just
```julia
function loglikelihood(m::LinearModel)
    n = nobs(m)
    -n/2 * (log(2π * deviance(m)/n) + 1)
end

function nullloglikelihood(m::LinearModel)
    n = nobs(m)
    -n/2 * (log(2π * nulldeviance(m)/n) + 1)
end
```

Reply:
thanks for finding this. It works (when `n` is the sum of weights).